packages/sdk/test/e2e/browser/teamsUserCredential.browser.spec.ts

```diff
@@ -36,28 +36,53 @@ describe("TeamsUserCredential Tests - Browser", () => {
         });
       });
   });
-  after(() => {
+  afterEach(() => {
     sinon.restore();
   });
 
   it("GetUserInfo should success with SSOToken", async function () {
     const credential: TeamsUserCredential = new TeamsUserCredential();
     const info = await credential.getUserInfo();
     assert.strictEqual(info.preferredUserName, env.SDK_INTEGRATION_TEST_ACCOUNT_NAME);
-    assert.strictEqual(info.displayName, env.SDK_INTEGRATION_TEST_ACCOUNT_NAME.split("@", 1)[0]);
+    assert.strictEqual(info.displayName, "Integration Test");
     assert.strictEqual(info.objectId, TEST_USER_OBJECT_ID);
   });
 
   it("GetToken should success with consent scope", async function () {
+    sinon.stub(PublicClientApplication.prototype, <any>"getAccountByUsername").returns(null);
+    sinon
+      .stub(PublicClientApplication.prototype, <any>"acquireTokenSilent")
+      .callsFake(async (): Promise<AuthenticationResult> => {
+        const graphToken = await getGraphToken(ssoToken, "User.Read");
+        return new Promise((resolve) => {
+          resolve({
+            authority: "authority",
+            uniqueId: "uniqueId",
+            tenantId: "tenantId",
+            scopes: ["User.Read"],
+            account: null,
+            idToken: "idToken",
+            idTokenClaims: {},
+            accessToken: graphToken,
+            fromCache: false,
+            expiresOn: null,
+            tokenType: "tokenType",
+            correlationId: "correlationId",
+          });
+        });
+      });
+
     const credential: TeamsUserCredential = new TeamsUserCredential();
     // await expect(credential.getToken(["User.Read"])).to.be.eventually.have.property("token");
     const accessToken = await credential.getToken(["User.Read"]);
     const decodedToken = jwtDecode<AADJwtPayLoad>(accessToken!.token);
     assert.strictEqual(decodedToken.aud, "00000003-0000-0000-c000-000000000000");
-    assert.isTrue(decodedToken.scp!.startsWith("User.Read"));
+    assert.isTrue(decodedToken.scp!.includes("User.Read"));
   });
 
   it("GetToken should throw UiRequiredError with unconsent scope", async function () {
+    sinon.stub(PublicClientApplication.prototype, <any>"acquireTokenSilent").throws("error");
```

Comment:
what will happen if we don't mock these 2 errors?

Reply:
Here these two are added to shorten the test time. It is acceptable that the test takes 10-20 seconds longer, thus I removed them. Thx!